types/jsforce/record-stream.d.ts

```diff
@@ -0,0 +1,7 @@
+import { Transform } from 'stream';
+
+export class RecordStream extends Transform {
+  map(): Transform;
```

Comment:
It looks like the definition of these functions have different signatures [here](https://jsforce.github.io/jsforce/doc/record-stream.js.html)?

Reply:
Good catch, they require arguments

```js
filter(fn)
map(fn)
```

This needs fixing

Also, RecordStream needs static methods, which is not included here

```
recordMapStream(record, noevalopt)
map(fn)
filter(fn)
```

And I cannot see stream() in the API